stable/sentry/templates/hooks/db-init.job.yaml

```diff
@@ -1,4 +1,5 @@
 # https://docs.sentry.io/server/installation/docker/#running-migrations
+{{- if .Values.hooks.dbInit.enabled -}}
```

Comment:
@max-rocket-internet could you remove the second one dash?
It seems it brokes template:
```
---
# Source: sentry/templates/hooks/db-init.job.yaml
# https://docs.sentry.io/server/installation/docker/#running-migrationsapiVersion: batch/v1
kind: Job
```

https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/helm_charts/20293/pull-charts-e2e/1246833451739385856/#1:build-log.txt%3A202

Reply:
OK done!